packages/mds-api-server/tests/index.spec.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
```

Comment:
So, `mocha` tests can be converted to use async/await instead of `done()` callbacks. Not worth it?

Reply:
They can, but it's a pretty huge pain -- I think it makes more sense to just write tests using async/await moving forward. Much less crucial than having async/await for core packages IMO.